proto/executionapis/astria/execution/v1/execution.proto

```diff
@@ -14,9 +14,15 @@ message GenesisInfo {
   // The rollup_id is the unique identifier for the rollup chain.
   astria.primitive.v1.RollupId rollup_id = 1;
   // The first block height of sequencer chain to use for rollup transactions.
-  uint32 sequencer_genesis_block_height = 2;
+  uint32 sequencer_start_block_height = 2;
```

Comment:
Does this warrant a breaking change?

Reply:
I don't believe this would be a breaking change (even though the proto breaking changes lint would disagree), since the field is identified by the number itself as opposed to the name. The name change is just to more accurately reflect the new functionality, since conductor can now start from anywhere, not just genesis. I do think the proto changes at large are effectively breaking though, since the old shape will no longer work with with conductor (`sequencer_stop _height` should default to 0 in that case, in which case it will never execute any blocks).

Reply:
You are right that the name is indeed a better reflection of how this field is used.

However, I would in general be more conservative when changing APIs: a breaking change (in this case, in the JSON serialization) weighs heavier than a name that's no longer a completely honest representation of the functionality.